service/src/test/resources/ehr/mapper/observation/example-observation-resource-13.json

```diff
@@ -0,0 +1,27 @@
+{
+  "resourceType":"Observation",
+  "id":"Consultation3-Topic2-Category-Examination-Observation-1",
+  "effectiveDateTime":"2019-06-01T10:30:00+00:00",
+  "issued":"2019-06-01T10:30:00+00:00",
+  "comment": "Test comment",
+  "referenceRange":[
+    {
+      "text":"Test reference range text",
+      "high":{
+        "value":20
+      }
+    }
+  ],
+  "code": {
+    "text": "test"
+  },
+  "interpretation": {
+    "coding": [
+      {
+        "system": "http://hl7.org/fhir/v2/0078",
+        "code": "Example",
+        "display": "Example without system"
+      }
```

Comment:
are we missing `"userSelected": true` here? or is this a different test scenario?

Reply:
Makes sense, thanks for clarifying 👍

Reply:
I'll add another test case to clarify this 😄

Reply:
Even better 😄

Reply:
Added another test case to prove that `"userSelected": true` is prioritise over other cases where `userSelected` is / isn't involved here 254de75 

I'd like to add another test case to prove that if `isInterpretationCode` is true, then this coding is prioritised, but I'll discuss that with you in our call later on.

Reply:
Created new test code to prove that `isInterpretationCode` is `true`, then translating that to `InterpretationCode` is prioritised over other codings that do not meet `isInterpretationCode` in e3640d2

Reply:
Looks good, thanks for creating that